frontend/src/pages/CourseSelector/CourseSelector.jsx

```diff
@@ -17,6 +17,25 @@ const CourseSelector = () => {
     programCode, programName, specialisation, minor,
   } = useSelector((state) => state.degree);
 
+  const { courses } = useSelector((state) => state.planner);
+
+  useEffect(() => {
+    const openNotification = () => {
+      notification.info({
+        message: "How do I unlock courses on the sidebar?",
+        description: "Quick add courses are made visible when you add their prerequisites to your planner and fulfill UOC requirements. Otherwise all courses can be viewed via the search bar.",
```

Comment:
lmao i feel like the message notification is a bit long and takes up more room based on the image haha. Perhaps you could shorten it? If not, its okay!

Reply:
ahaha yeah sure I'll try shorten it

Reply:
<img width="1420" alt="image" src="https://user-images.githubusercontent.com/90736577/168413590-35936494-ca30-4f9e-b5a1-964e2ac7733b.png">

Reply:
changed it to this - lemme know if it's g :)

Reply:
hmm maybe say "courses are shown as you meet the requirements to take them. Courses can also be... "

Reply:
Yeahh that's much more succinct, i'll probably use this "Courses are shown as you meet the requirements to take them. Any course can also be selected via the search bar."